src/test/scala/io/github/mkotsur/firebase/rest/FirebaseClientTest.scala

```diff
@@ -138,6 +160,22 @@ class FirebaseClientTest extends FunSpec with Matchers with ScalaFutures with Tr
           fc.get[Int](s"temp/pushed/$pushedChildName").futureValue shouldBe Some(43)
           fc.delete("temp").futureValue shouldBe ((): Unit)
         }
+        
+      it("should update the values of an existing list into Firebase") {
+        val adminCredential = AdminCredentials(validJsonKey)
+        implicit val token = FirebaseClient.getToken(adminCredential).success.value
+        val fc = new FirebaseClient(projectId)
+
+        fc.put[List[String]](List("apple", "cherry"), "listsEverywhere/listsOfFruits")
+        Thread.sleep(1000)
+        val currentFruits = fc.get[List[String]]("listsEverywhere/listsOfFruits").futureValue
+        currentFruits shouldBe Some(List("apple", "cherry"))
+
+        fc.put[List[String]](currentFruits.get ::: List[String]("pear"), "listsEverywhere/listsOfFruits")
```

Comment:
Codacy doesn't seem to be completely happy with `.get` on option. Fortunatelly there is a special trait `OptionValues` for this in `ScalaTest`! Add it to the class and you'll be able to do `.value`, and Scalatest will make much nicer error report if there is no value.